index.html

```diff
@@ -9,11 +9,71 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <!-- header starts here-->
+<header id="header">  
+   <nav id="nav-bar"> 
+     <img src= "img\karma-logo.svg" alt="Karma">
+     <ul class="nav">
+       <li> <a class="nav-link" href="#MeetKarma"> Meet Karma </a></li>
+       <li> <a class="nav-link" href="#Howitworks"> How it works </a> </li>   
+       <li> <a class="nav-link" href="#Store"> Store </a> </li> 
+       <li> <a class="nav-link" href="#Blog"> Blog </a> </li>
+       <li> <a class="nav-link" href="#Help"> Help </a></li>
+       <li> <a class="nav-link" href="#Login"> Login</a></li> 
+     </ul>
+  </nav> 
+</header> <!--header ends here-->
+
+<article>
+   <!--Introduction starts here-->
+   <section id="intro"> 
+      <div id="intro-content">
+       <h1> Introducing Karma </h1>
+       <h2> Bring WiFi with you, everywhere you go.</h2>
+       <button type="button" id="learnbutton"> Learn More </button>
+     </div>
+   </section>
+   <!-- Introduction ends here-->
+
+   <!--content starts here-->
+   <section id="content"> 
+      <h1> Everyone needs a little Karma.</h1> 
```

Comment:
```suggestion
      <h2> Everyone needs a little Karma.</h2> 
```
We should only have one `h1` on the page, so we should update this to `h2`